apps/design-land/src/app/modal/modal.module.ts

```diff
@@ -1,24 +1,31 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { 
+  DaffModalModule, 
+  DaffFeatureModule, 
+  DaffButtonModule,
+  DaffCardModule 
+} from '@daffodil/design';
 
 import { ModalComponent } from './modal.component';
-import { DaffModalModule } from '@daffodil/design';
-import { DaffFeatureModule } from '@daffodil/design';
-import { DaffButtonModule } from '@daffodil/design';
-
+import { DesignLandModalContentComponent } from './modal-content/modal-content.component';
 import { DesignLandModalRoutingModule } from './modal-routing.module';
 
 @NgModule({
   declarations: [
     ModalComponent,
+    DesignLandModalContentComponent,
   ],
   imports: [
     CommonModule,
     DesignLandModalRoutingModule,
     DaffModalModule,
     DaffButtonModule,
-    DaffFeatureModule
-
+    DaffFeatureModule,
+    DaffCardModule
+  ],
+  entryComponents: [
+    DesignLandModalContentComponent
   ]
 })
 export class ModalModule { 
```

Comment:
Prefix with `DesignLand` ?

Reply:
fixed in cf659493bb6334ae3b97c3125b3d6f4f6ab27cef